elixir/apps/domain/lib/domain/auth/adapters/microsoft_entra.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Domain.Auth.Adapters.MicrosoftEntra do
+  use Supervisor
+  alias Domain.Actors
+  alias Domain.Auth.{Provider, Adapter}
+  alias Domain.Auth.Adapters.OpenIDConnect
+  alias Domain.Auth.Adapters.MicrosoftEntra
+  require Logger
+
+  @behaviour Adapter
+  @behaviour Adapter.IdP
+
+  def start_link(_init_arg) do
+    Supervisor.start_link(__MODULE__, nil, name: __MODULE__)
+  end
+
+  @impl true
+  def init(_init_arg) do
+    children = [
+      MicrosoftEntra.APIClient,
+      {Domain.Jobs, MicrosoftEntra.Jobs}
+    ]
+
+    Supervisor.init(children, strategy: :one_for_one)
+  end
+
+  @impl true
+  def capabilities do
+    [
+      provisioners: [:just_in_time, :custom],
```

Comment:
Please remove JIT from this list, I'll do that in other places in a different PR.

Reply:
Removed the `:just_in_time` option.